.gitignore

```diff
@@ -146,4 +146,7 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-cache
\ No newline at end of file
+cache
+
+#temp files
+tempCodeRunnerFile.py
```

Comment:
Revert the change in this file. If you want to have some file you just should not commit it